### ISSUE TITLE
New skins for Android

### ISSUE THREAD

**Issue by fabriziomoscon** [underscopeio/react-native-facebook-account-kit#65]
It looks like Facebook added new skin styles: classic, translucent, contemporary.
Because this library uses `compile ('com.facebook.android:account-kit-sdk:4.+')` my app adopted the new skin style as soon as I recompiled.
It would be great to add the possibility to pick either skin from the configuration.

### Additional Information

* React Native Account Kit version: 0.6.2
* React Native version: 0.41.2
* Platform: Android

**Comment by gaguirre** [comment]
Yeah, it would be great! I'm kind of busy right now, so a PR would be even greater 😃 
Otherwise you'll need to wait until next week probably.

**Comment by fabriziomoscon** [comment]
It is not a stopper for me, I just thought that in case somebody would ask it was a good idea to mention the issue in a ticket.
In case I found some issue with the current code and new skins I will report.
Cheers

**Comment by pewh** [comment]
any news about this?

**Comment by gaguirre** [comment]
@pewh I had no time to tackle this yet.
Any contribution is welcome ;)

**Comment by giladno** [comment]
It seems that you don't have to change your xml for Android anymore, but can do it directly from Java (https://developers.facebook.com/docs/accountkit/android/customizing). Would be nice to have a single JS interface for applying the theme for both platforms.

**Comment by gaguirre** [comment]
@giladno totally agree about keeping the same JS interface for Android and iOS.
I've no time these days to tackle this, maybe in two weeks... any PR is welcome.

**Comment by giladno** [comment]
@gaguirre I wish I could help, but I'm not an Android developer, I can only with with ObjC code :)
